src/Aspire.Dashboard/Model/DashboardClient.cs

```diff
@@ -98,6 +100,28 @@ GrpcChannel CreateChannel()
                 KeepAlivePingPolicy = HttpKeepAlivePingPolicy.WithActiveRequests
             };
 
+            var authMode = configuration.GetEnum<ResourceClientAuthMode>("ResourceServiceClient:AuthMode");
```

Comment:
```suggestion
            var authMode = configuration.GetEnum<ResourceClientAuthMode>("Dashboard:ResourceServiceClient:AuthMode");
```

Reply:
What do you think about changing `ResourceServiceClient` to `ResourcesClient`. It's a bit shorter.

Reply:
The "Resource Service" is a concept that exists in the system. This is a client for that concept. I don't think shortening it is worth the ambiguity.